mono/metadata/w32file-unix.c

```diff
@@ -1947,6 +1949,7 @@ mono_w32file_create(const gunichar2 *name, guint32 fileaccess, guint32 sharemode
 	gchar *filename;
 	gint fd, ret;
 	struct stat statbuf;
+	ERROR_DECL (err);
```

Comment:
nit: naming - we usually call the variable `error`. (here an in a few other places)

Reply:
It seemed inconsistent so I just picked one - will switch over.